src/components/catalog/ShopNavComponentNew.tsx

```diff
@@ -48,7 +53,8 @@ export const ShopNavComponentNew = ({ handleFilterNav }: any) => {
         </div>
         <ul className="agri-ul widget-wrapper">
           <li>
-            <a href="#" className="d-flex flex-wrap justify-content-between">
+            <a href="#" className="d-flex flex-wrap justify-content-between"
```

Comment:
We try to replace any href="#" for type="button"

Reply:
this href is needed for travell top site to user when him clicks it